### ISSUE TITLE
a11y: aria-expanded used where it is not allowed

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-accordion#165]
### What browser client(s) are you seeing the issue on?

- [x] Firefox
- [x] Chrome
- [x] Safari
- [x] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [x] Desktop
- [x] Mobile

## See code 

https://github.com/AlaskaAirlines/auro-accordion/blob/c2d5040aba11ed93611a12752859dec869a3c812/src/auro-accordion.js#L197-L223

## This problem 

One or more elements **are** using the `aria-expanded` attribute where it is not allowed. When the `aria-expanded` attribute is not permitted and no appropriate element or ARIA role is available, the attribute must be removed. If the element with the `aria-expanded` attribute controls the expansion of another grouping container that it does not "own," the author must reference the container using the `aria-controls` attribute.  

![Image](https://github.com/user-attachments/assets/2c7476d1-ba8a-4972-b7cd-6e645153e345)

## Recommended solution

FWIW, you CANNOT use `aria-controls` on the outer `auro-accordion-button` tag due to shadow DOM constraints. In order to make that work, `CustomEvent` functionality would need to be developed. 

Preferably, `aria-expanded` must be directly on the `button` inside the shadow DOM.

```html
<button id="accordionButton" aria-expanded="true">
```

Another recommendation is to consider using `aria-controls="accordionContent"` on the `<auro-accordion-button>` element and then adding `id="accordionContent"` to the `div` with `part="content"`. These two elements are within the same `<auro-accordion>` element shadow DOM, so should be in the same tree and pass accessibility testing. 

## Additional context

This issue was initially reported via the following issue that was recently closed 

- https://github.com/AlaskaAirlines/auro-accordion/issues/161

### Expected behavior

Assistive ARIA technology should be used in compliance with the specification to be rendered and communicated clearly to a user with assistive technologies. 

### Which version of auro-templates do you have installed?

@current

### Reproducible on the Auro doc site

- [x] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by blackfalcon** [comment]
Code updated on Auro docsite and validated and corrected. 

I know why I was seeing the error persist, the element requires a full clean build due to cached assets when running locally.
